pkg/asset/installconfig/powervs/validation_test.go

```diff
@@ -608,18 +608,21 @@ func TestValidateSystemTypeForZone(t *testing.T) {
 			},
 			errorMsg: fmt.Sprintf("%s is not available in: %s", newSysType, validZone),
 		},
-		{
-			name: "Good Zone/SysType combo specified",
-			edits: editFunctions{
-				func(ic *types.InstallConfig) {
-					ic.Platform.PowerVS.Region = validRegion
-					ic.Platform.PowerVS.Zone = validZone
-					ic.ControlPlane.Platform.PowerVS = validMachinePool()
-					ic.ControlPlane.Platform.PowerVS.SysType = defaultSysType
+		/*
```

Comment:
Do you still need this code that is in comments?

Reply:
Hrm. I could remove it.  I don't think we can mock this.